core/src/main/scala/org/locationtech/rasterframes/expressions/generators/RasterSourceToRasterRefs.scala

```diff
@@ -77,9 +76,9 @@ case class RasterSourceToRasterRefs(children: Seq[Expression], bandIndexes: Seq[
     }
     catch {
       case NonFatal(ex) ⇒
-        val payload = Try(children.map(c => RasterSourceType.deserialize(c.eval(input)))).toOption.toSeq.flatten
-        logger.error("Error fetching data for one of: " + payload.mkString(", "), ex)
-        Traversable.empty
+        val description = Try(children.map(c => RasterSourceType.deserialize(c.eval(input))))
+          .toOption.toSeq.flatten.mkString(", ")
```

Comment:
Hmm we have lost the pretty useful text of "Error fetching data for one of" 

Recommend reinstating that.

Reply:
Ooops. Not intended. Good catch.